ambry-api/src/main/java/com.github.ambry/clustermap/PartitionStateChangeListener.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.clustermap;
+
+/**
+ * {@link PartitionStateChangeListener} takes action when partition state changes.
+ */
+public interface PartitionStateChangeListener {
+
+  /**
+   * Action to take on becoming leader of a partition.
+   * @param partitionName of the partition.
+   */
+  void onPartitionLeadFromStandby(String partitionName);
```

Comment:
Could you change the name to `onPartitionBecomeLeaderFromStandby` ? I feel like this is more accurate.